src/test/scala/org/broadinstitute/dsde/firecloud/service/NihServiceUnitSpec.scala

```diff
@@ -206,7 +229,7 @@ class NihServiceUnitSpec extends AnyFlatSpec with Matchers with BeforeAndAfterEa
   }
 
   private def verifyTargetGroupSynced(): Unit = {
-    val emailsToSync = Set(WorkbenchEmail(userTcgaAndTarget.email.value), WorkbenchEmail(userTargetOnly.email.value))
+    val emailsToSync = Set(userTcgaAndTarget.email, userTargetOnly.email)
```

Comment:
Was there a reason to pull out the value of the email and then turn it back into a `WorkbenchEmail`?

Reply:
probably not, it it was probably an instance of `RawlsEmail` or something once upon a time